libp2p/muxers/mplex/mplex.nim

```diff
@@ -68,19 +68,19 @@ proc cleanupChann(m: Mplex, chann: LPChannel) {.async, inline.} =
       libp2p_mplex_channels.set(
         m.channels[chann.initiator].len.int64,
         labelValues = [$chann.initiator, $m.connection.peerId])
-  except CatchableError as exc:
+  except CancelledError as exc:
```

Comment:
can't `m.channels[chann.initiator]` raise `KeyError`?

Reply:
no, `m.channels` is an `array[bool, ...]`, not a `Table`, so both `false` and `true` always exist.